c/include/foxglove-c/foxglove-c.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * NOTE: This file is autogenerated by cbindgen.
+ *
+ * Foxglove SDK
+ * https://github.com/foxglove/foxglove-sdk
+ */
+
+
+#ifndef FOXGLOVE_C_H
+#define FOXGLOVE_C_H
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+typedef struct foxglove_websocket_server foxglove_websocket_server;
+
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+/**
+ * Create and start a server. The server must later be freed with `foxglove_server_destroy`.
+ *
+ * # Safety
+ * `name` and `host` must be null-terminated strings with valid UTF8.
+ */
+struct foxglove_websocket_server *foxglove_server_start(const char *name,
```

Comment:
This will quickly get unwieldy, it would be better to pass in a C version of the ServerOptions struct. Don't have to address it now.

Maybe call this foxglove_start_server (verb first)